CONTRIBUTING.md

```diff
@@ -151,7 +151,7 @@ repository](https://github.com/zulip/zulip/issues?q=is%3Aopen+is%3Aissue+label%3
 Post a comment with `@zulipbot claim` to
 the issue thread. [Zulipbot](https://github.com/zulip/zulipbot) is a GitHub
 workflow bot; it will assign you to the issue and label the issue as "in
-progress". You can only claim issues with the
+progress". Before making your first pull request, you can only claim issues with the
```

Comment:
It's not until you've made your first pull request, it's until your first commit is merged.

@alya FYI for wording thoughts on this.

Reply:
@timabbott I've made the changes suggested by @alya, although I'm still a little confused at what exactly the restrictions for claiming an issue are. See my [last comment](https://github.com/zulip/zulip/pull/20891#issuecomment-1021951023) for context.